src/js/view/controls/controlbar.js

```diff
@@ -441,8 +441,12 @@ export default class Controlbar {
 
     // Close menus if it has no event.  Otherwise close all but the event's target.
     closeMenus(evt) {
+        if (evt) {
+            return;
+        }
+
         this.menus.forEach(ele => {
-            if (!evt || evt.target !== ele.el) {
+            if (evt.target !== ele.el) {
                 ele.closeTooltip(evt);
```

Comment:
@waxidiotic is there ever a case where ele is undefined ? Do we ever add undefined or null elements to the menu array ?